Rollout/launch checklist.md

```diff
@@ -136,11 +136,11 @@ Hear from all stakeholders how things are going so far, what concerns may be bub
 - [ ] Chris decides whether to continue or roll back
 - [ ] Chris disseminates decision
 
-### 2:00 PM ET: Monitor
+### 8:00 PM ET: Monitor
 - [ ] Enact Vets.gov redirects
 - [ ] Full team watches monitoring dashboard, google analytics, social media buzz
 
-### 5:00 PM ET: end of day check-in
+### 12:00 AM ET (Nov 8): end of day check-in
```

Comment:
Let's move this to 10PM Eastern.  We should be able to tell in 2 hours how things are going.